### ISSUE TITLE
Support contentEditable elements.

### ISSUE THREAD

**Issue by tmlangley** [gabesullice/vimsert#4]
Recommended by @smblott-github.

**Comment by gabesullice** [comment]
I haven't looked much into this, but I have some off the cuff concerns. Given the HTML below:

``` html
<div contentEditable="true">
  <ul>
    <li>Some Content</li>
    <li>Other Content</li>
  </ul>
</div>
```

How would vimsert know which element has focus? Would you only be able to edit one `<li>` at a time, or all of the `div` element? If the whole div, how would we address the fact that the elements themselves aren't editable?

**Comment by tmlangley** [comment]
Given the fact that most vim users are probably developers of some sort, would it make sense to show the actual html in the vimsert popup? I know I would personally prefer that.

**Comment by gabesullice** [comment]
I agree, it would be nice to see the HTML, however, as I understand it, it
is not possible to change anything about the elements. So, I think it would
be very difficult to prevent editing the elements.

**Comment by tmlangley** [comment]
I think we would just copy the innerHTML into the vimsert popup and then replace it just like we do with the textarea value now.

**Comment by gabesullice** [comment]
Okay, so we would only pull the content of the current element in focus,
not the whole contentEditable div?

**Comment by smblott-github** [comment]
Yes.  I was going to make the same suggestion. if `document.activeElement.isContentEditable` is truthy, the grab `document.activeElement.innerHTML`.  This works.  Not sure what happens when you later write it back, but I expect it works.

(Separately, I think you need a `keypress` listener or `window`.  You can't always detect what's content editable statically.  That's what we found over at Vimium.)
